x-pack/platform/plugins/shared/security/server/authentication/providers/base.ts

```diff
@@ -81,6 +81,10 @@ export abstract class BaseAuthenticationProvider {
     this.logger = options.logger;
   }
 
+  doesSessionNeedToBeCheckedForRequestIds(state?: unknown): boolean {
```

Comment:
__nit:__ it should be automatically inferred.
```suggestion
  doesSessionNeedToBeCheckedForRequestIds(state?: unknown) {
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/220426/commits/413cced82b12c201040f71876028729606ebd92c